### ISSUE TITLE
JPG preview not working

### ISSUE THREAD

**Issue by pojntfx** [flathub/net.scribus.Scribus#8]
While using this version of Scribus, I've encountered a bug: It can't preview JPG images.

See these screenshots:

![Screenshot from 2019-06-14 10-44-09](https://user-images.githubusercontent.com/28832235/59496468-c8191580-8e91-11e9-8f44-58096c700eb2.png)

![Screenshot from 2019-06-14 10-44-49](https://user-images.githubusercontent.com/28832235/59496472-cb140600-8e91-11e9-95e5-a672ed761fbb.png)

It works for PNGs, PDFs etc., just JPGs are the problem. The dialog above it the one that appears after right-clicking an image frame and then  `Content/Get Image`.

My system:

```bash
NAME=Fedora
VERSION="30.20190610.0 (Workstation Edition)"
ID=fedora
VERSION_ID=30
VERSION_CODENAME=""
PLATFORM_ID="platform:f30"
PRETTY_NAME="Fedora 30.20190610.0 (Workstation Edition)"
ANSI_COLOR="0;34"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:30"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f30/system-administrators-guide/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=30
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=30
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
OSTREE_VERSION=30.20190610.0
```

Thanks for any help!

**Comment by jurf** [comment]
Hey, sorry for the really late reply. Does this still happen? I believe 1.5.5 was released in the meantime.

**Comment by pojntfx** [comment]
@jurf Thanks for your reply!

Sorry, I can't test ATM - I'm currently using a Mac.

**Comment by aarproducoes** [comment]
@jurf still happen on my system.

OS: Pop!_OS 20.04 LTS x86_64
Kernel: 5.8.0-7625-generic
Shell: bash 5.0.17
Resolution: 1920x1080
DE: Gnome
WM: Mutter
CPU: Intel i7-4810MQ (8) @ 3.800GHz
GPU: Intel 4th Gen Core Processor
GPU: NVIDIA GeForce GT 730M
Memory: 1881MiB / 15931MiB
/home=Xfs (where are .jpg files)

Thanks!

**Comment by jurf** [comment]
I quickly checked the [Arch package](https://www.archlinux.org/packages/community/x86_64/scribus/) – I think we’re missing `libjpeg`. I don’t have time to spare ATM, but it should be a 30-minute fix if someone’s up to it.

**Comment by hfiguiere** [comment]
On the current flatpak JPEG don't even appear in the file dialog.

**Comment by jurf** [comment]
Yeah, it’s a nuisance, but IIRC it should work in Scribus itself.

**Comment by PaulchenPlump** [comment]
Unfortunately, the problem still exists. Also, the import and save dialog does not show the available formats (see screenshots of the thread opener). Under a native Archlinux installation everything is correct!

**Comment by travier** [comment]
We apparently have `libjpeg` in the runtime so this could be something else. Not sure what's happening here.

**Comment by hfiguiere** [comment]
```
-- Found JPEG: /usr/lib/x86_64-linux-gnu/libjpeg.so (found version "62") 
JPEG Library Found OK
```

So it claims it's ok.

**Comment by almereyda** [comment]
Upstream this is https://bugs.scribus.net/view.php?id=17199

**Comment by belfasthenry** [comment]
> Upstream this is https://bugs.scribus.net/view.php?id=17199

Which is closed, "Can't reproduce". The opinion there is that it's a specific problem with the Flatpak.
